service/integration/keymanagement_test.go

```diff
@@ -478,6 +708,10 @@ func (s *KeyManagementSuite) deleteTestProviderConfigs(ids []string) {
 	}
 }
 
+func (s *KeyManagementSuite) getUniqueProviderName(baseName string) string {
```

Comment:
What are the rules for this? you could also append the test name with t.Name that is unique per run of the suite